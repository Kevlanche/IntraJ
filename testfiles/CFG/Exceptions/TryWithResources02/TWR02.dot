digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[8]"}
{ rank = same; "SingleTypeImportDecl[4]"; "SingleTypeImportDecl[6]"; "ClassDecl[9]"}
{ rank = same; "TypeAccess[5]"; "TypeAccess[7]"; "Modifiers[10]"; "Opt[12]"; "List[13]"; "List[14]"}
{ rank = same; "List[11]"; "MethodDecl[15]"}
{ rank = same; "Entry[16]"; "Modifiers[17]"; "PrimitiveTypeAccess[19]"; "List[20]"; "List[21]"; "Opt[22]"; "Exit[23]"}
{ rank = same; "List[18]"; "Block[23]"}
{ rank = same; "List[24]"}
{ rank = same; "TryWithResources[25]"}
{ rank = same; "List[26]"; "Block[44]"; "List[58]"; "Opt[74]"; "UncheckedExceptions[75]"}
{ rank = same; "ResourceDeclaration[27]"; "List[45]"; "BasicCatch[59]"}
{ rank = same; "List[33]"; "Opt[34]"; "ResourceModifiers[41]"; "TypeAccess[43]"; "MethodAccess[28]"; "ExprStmt[46]"; "ParameterDeclaration[60]"; "Block[64]"}
{ rank = same; "ClassInstanceExpr[35]"; "List[42]"; "List[30]"; "List[31]"; "List[32]"; "UncheckedExceptions[29]"; "Dot[47]"; "Modifiers[61]"; "TypeAccess[63]"; "List[65]"}
{ rank = same; "TypeAccess[36]"; "List[37]"; "Opt[38]"; "List[39]"; "List[40]"; "VarAccess[48]"; "MethodAccess[49]"; "List[62]"; "ExprStmt[66]"}
{ rank = same; "List[51]"; "List[52]"; "List[57]"; "UncheckedExceptions[50]"; "Dot[67]"}
{ rank = same; "NTACloseResource[53]"; "VarAccess[68]"; "MethodAccess[69]"}
{ rank = same; "List[54]"; "List[55]"; "List[56]"; "List[71]"; "List[72]"; "List[73]"; "UncheckedExceptions[70]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"SingleTypeImportDecl[4]" [label="SingleTypeImportDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[5]" [label="TypeAccess\n < ByteArrayOutputStream >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"SingleTypeImportDecl[6]" [label="SingleTypeImportDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[7]" [label="TypeAccess\n < IOException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[8]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[9]" [label="ClassDecl\n < TWR02 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[10]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[12]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[13]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[14]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[15]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[16]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[17]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[19]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[21]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[22]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[23]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryWithResources[25]" [label="TryWithResources\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ResourceDeclaration[27]" [label="ResourceDeclaration\n < ByteArrayOutputStream baos >", fillcolor=white   style=filled  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[34]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassInstanceExpr[35]" [label="ClassInstanceExpr\n <ByteArrayOutputStream>", fillcolor=white   style=filled  ]
"TypeAccess[36]" [label="TypeAccess\n < ByteArrayOutputStream >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[38]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[39]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ResourceModifiers[41]" [label="ResourceModifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[42]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[43]" [label="TypeAccess\n < ByteArrayOutputStream >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[28]" [label="MethodAccess\n < close >", fillcolor=white   style=filled  ]
"List[30]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[31]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[29]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"Block[44]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[45]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[46]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[47]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[48]" [label="VarAccess\n < baos >", fillcolor=white   style=filled  ]
"MethodAccess[49]" [label="MethodAccess\n < flush >", fillcolor=white   style=filled  ]
"List[51]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[52]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"NTACloseResource[53]" [label="NTACloseResource\n <baos.close>", fillcolor=white   style=filled  ]
"List[54]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[56]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[50]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[58]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BasicCatch[59]" [label="BasicCatch\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[60]" [label="ParameterDeclaration\n < e >", fillcolor=white   style=filled  ]
"Modifiers[61]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[62]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[63]" [label="TypeAccess\n < IOException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[64]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[66]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[67]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[68]" [label="VarAccess\n < e >", fillcolor=white   style=filled  ]
"MethodAccess[69]" [label="MethodAccess\n < printStackTrace >", fillcolor=white   style=filled  ]
"List[71]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[72]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[73]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[70]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[74]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[75]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"Exit[23]" [label="Exit\n ", fillcolor=white   style=filled  ]
"SingleTypeImportDecl[4]" -> "TypeAccess[5]" [style=dashed, color=gray]
"List[3]" -> "SingleTypeImportDecl[4]" [style=dashed, color=gray]
"SingleTypeImportDecl[6]" -> "TypeAccess[7]" [style=dashed, color=gray]
"List[3]" -> "SingleTypeImportDecl[6]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"Modifiers[10]" -> "List[11]" [style=dashed, color=gray]
"ClassDecl[9]" -> "Modifiers[10]" [style=dashed, color=gray]
"ClassDecl[9]" -> "Opt[12]" [style=dashed, color=gray]
"ClassDecl[9]" -> "List[13]" [style=dashed, color=gray]
"MethodDecl[15]" -> "Entry[16]" [style=dashed, color=gray]
"Modifiers[17]" -> "List[18]" [style=dashed, color=gray]
"MethodDecl[15]" -> "Modifiers[17]" [style=dashed, color=gray]
"MethodDecl[15]" -> "PrimitiveTypeAccess[19]" [style=dashed, color=gray]
"MethodDecl[15]" -> "List[20]" [style=dashed, color=gray]
"MethodDecl[15]" -> "List[21]" [style=dashed, color=gray]
"ResourceDeclaration[27]" -> "List[33]" [style=dashed, color=gray]
"ClassInstanceExpr[35]" -> "TypeAccess[36]" [style=dashed, color=gray]
"ClassInstanceExpr[35]" -> "List[37]" [style=dashed, color=gray]
"ClassInstanceExpr[35]" -> "Opt[38]" [style=dashed, color=gray]
"ClassInstanceExpr[35]" -> "List[39]" [style=dashed, color=gray]
"ClassInstanceExpr[35]" -> "List[40]" [style=dashed, color=gray]
"Opt[34]" -> "ClassInstanceExpr[35]" [style=dashed, color=gray]
"ResourceDeclaration[27]" -> "Opt[34]" [style=dashed, color=gray]
"ResourceModifiers[41]" -> "List[42]" [style=dashed, color=gray]
"ResourceDeclaration[27]" -> "ResourceModifiers[41]" [style=dashed, color=gray]
"ResourceDeclaration[27]" -> "TypeAccess[43]" [style=dashed, color=gray]
"MethodAccess[28]" -> "List[30]" [style=dashed, color=gray]
"MethodAccess[28]" -> "List[31]" [style=dashed, color=gray]
"MethodAccess[28]" -> "List[32]" [style=dashed, color=gray]
"MethodAccess[28]" -> "UncheckedExceptions[29]" [style=dashed, color=gray]
"ResourceDeclaration[27]" -> "MethodAccess[28]" [style=dashed, color=gray]
"List[26]" -> "ResourceDeclaration[27]" [style=dashed, color=gray]
"TryWithResources[25]" -> "List[26]" [style=dashed, color=gray]
"Dot[47]" -> "VarAccess[48]" [style=dashed, color=gray]
"MethodAccess[49]" -> "List[51]" [style=dashed, color=gray]
"NTACloseResource[53]" -> "List[54]" [style=dashed, color=gray]
"NTACloseResource[53]" -> "List[55]" [style=dashed, color=gray]
"NTACloseResource[53]" -> "List[56]" [style=dashed, color=gray]
"List[52]" -> "NTACloseResource[53]" [style=dashed, color=gray]
"MethodAccess[49]" -> "List[52]" [style=dashed, color=gray]
"MethodAccess[49]" -> "List[57]" [style=dashed, color=gray]
"MethodAccess[49]" -> "UncheckedExceptions[50]" [style=dashed, color=gray]
"Dot[47]" -> "MethodAccess[49]" [style=dashed, color=gray]
"ExprStmt[46]" -> "Dot[47]" [style=dashed, color=gray]
"List[45]" -> "ExprStmt[46]" [style=dashed, color=gray]
"Block[44]" -> "List[45]" [style=dashed, color=gray]
"TryWithResources[25]" -> "Block[44]" [style=dashed, color=gray]
"Modifiers[61]" -> "List[62]" [style=dashed, color=gray]
"ParameterDeclaration[60]" -> "Modifiers[61]" [style=dashed, color=gray]
"ParameterDeclaration[60]" -> "TypeAccess[63]" [style=dashed, color=gray]
"BasicCatch[59]" -> "ParameterDeclaration[60]" [style=dashed, color=gray]
"Dot[67]" -> "VarAccess[68]" [style=dashed, color=gray]
"MethodAccess[69]" -> "List[71]" [style=dashed, color=gray]
"MethodAccess[69]" -> "List[72]" [style=dashed, color=gray]
"MethodAccess[69]" -> "List[73]" [style=dashed, color=gray]
"MethodAccess[69]" -> "UncheckedExceptions[70]" [style=dashed, color=gray]
"Dot[67]" -> "MethodAccess[69]" [style=dashed, color=gray]
"ExprStmt[66]" -> "Dot[67]" [style=dashed, color=gray]
"List[65]" -> "ExprStmt[66]" [style=dashed, color=gray]
"Block[64]" -> "List[65]" [style=dashed, color=gray]
"BasicCatch[59]" -> "Block[64]" [style=dashed, color=gray]
"List[58]" -> "BasicCatch[59]" [style=dashed, color=gray]
"TryWithResources[25]" -> "List[58]" [style=dashed, color=gray]
"TryWithResources[25]" -> "Opt[74]" [style=dashed, color=gray]
"TryWithResources[25]" -> "UncheckedExceptions[75]" [style=dashed, color=gray]
"List[24]" -> "TryWithResources[25]" [style=dashed, color=gray]
"Block[23]" -> "List[24]" [style=dashed, color=gray]
"Opt[22]" -> "Block[23]" [style=dashed, color=gray]
"MethodDecl[15]" -> "Opt[22]" [style=dashed, color=gray]
"MethodDecl[15]" -> "Exit[23]" [style=dashed, color=gray]
"List[14]" -> "MethodDecl[15]" [style=dashed, color=gray]
"ClassDecl[9]" -> "List[14]" [style=dashed, color=gray]
"List[8]" -> "ClassDecl[9]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[8]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[23]" -> "UncheckedExceptions[75]" [color=red, constraint=false]
"Exit[23]" -> "UncheckedExceptions[29]" [color=red, constraint=false]
"Exit[23]" -> "UncheckedExceptions[50]" [color=red, constraint=false]
"Exit[23]" -> "MethodAccess[69]" [color=red, constraint=false]
"UncheckedExceptions[75]" -> "Exit[23]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[75]" -> "MethodAccess[28]" [color=red, constraint=false]
"MethodAccess[69]" -> "Exit[23]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[69]" -> "VarAccess[68]" [color=red, constraint=false]
"VarAccess[68]" -> "MethodAccess[69]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[68]" -> "ParameterDeclaration[60]" [color=red, constraint=false]
"ParameterDeclaration[60]" -> "VarAccess[68]" [color=blue, constraint=false, xlabel=" "] 
"ParameterDeclaration[60]" -> "UncheckedExceptions[29]" [color=red, constraint=false]
"ParameterDeclaration[60]" -> "UncheckedExceptions[50]" [color=red, constraint=false]
"UncheckedExceptions[29]" -> "ParameterDeclaration[60]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[29]" -> "Exit[23]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[29]" -> "MethodAccess[28]" [color=red, constraint=false]
"MethodAccess[28]" -> "UncheckedExceptions[75]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[28]" -> "UncheckedExceptions[29]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[28]" -> "MethodAccess[49]" [color=red, constraint=false]
"UncheckedExceptions[50]" -> "ParameterDeclaration[60]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[50]" -> "Exit[23]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[50]" -> "NTACloseResource[53]" [color=red, constraint=false]
"NTACloseResource[53]" -> "UncheckedExceptions[50]" [color=blue, constraint=false, xlabel=" "] 
"NTACloseResource[53]" -> "MethodAccess[49]" [color=red, constraint=false]
"MethodAccess[49]" -> "MethodAccess[28]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[49]" -> "NTACloseResource[53]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[49]" -> "VarAccess[48]" [color=red, constraint=false]
"VarAccess[48]" -> "MethodAccess[49]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[48]" -> "ResourceDeclaration[27]" [color=red, constraint=false]
"ResourceDeclaration[27]" -> "VarAccess[48]" [color=blue, constraint=false, xlabel=" "] 
"ResourceDeclaration[27]" -> "ClassInstanceExpr[35]" [color=red, constraint=false]
"ClassInstanceExpr[35]" -> "ResourceDeclaration[27]" [color=blue, constraint=false, xlabel=" "] 
"ClassInstanceExpr[35]" -> "Entry[16]" [color=red, constraint=false]
"Entry[16]" -> "ClassInstanceExpr[35]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[8]" [style=invis]
"SingleTypeImportDecl[4]" -> "SingleTypeImportDecl[6]" [style=invis]
"Modifiers[10]" -> "Opt[12]" -> "List[13]" -> "List[14]" [style=invis]
"Entry[16]" -> "Modifiers[17]" -> "PrimitiveTypeAccess[19]" -> "List[20]" -> "List[21]" -> "Opt[22]" -> "Exit[23]" [style=invis]
"List[26]" -> "Block[44]" -> "List[58]" -> "Opt[74]" [style=invis]
"List[33]" -> "Opt[34]" -> "ResourceModifiers[41]" -> "TypeAccess[43]" [style=invis]
"TypeAccess[36]" -> "List[37]" -> "Opt[38]" [style=invis]
"VarAccess[48]" -> "MethodAccess[49]" [style=invis]
"ParameterDeclaration[60]" -> "Block[64]" [style=invis]
"Modifiers[61]" -> "TypeAccess[63]" [style=invis]
"VarAccess[68]" -> "MethodAccess[69]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
77", fillcolor=1]
Edges [label="Total Nr. Edges
110", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
14", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
17", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}