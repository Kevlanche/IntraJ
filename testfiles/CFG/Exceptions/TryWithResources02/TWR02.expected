NodeName: Entry[16] Successors: [ClassInstanceExpr[35]]
NodeName: ClassInstanceExpr[35] Successors: [ResourceDeclaration[27]]
NodeName: ResourceDeclaration[27] Successors: [VarAccess[48]]
NodeName: VarAccess[48] Successors: [MethodAccess[49]]
NodeName: MethodAccess[49] Successors: [MethodAccess[28], NTACloseResource[53]]
NodeName: MethodAccess[28] Successors: [UncheckedExceptions[75], UncheckedExceptions[29]]
NodeName: UncheckedExceptions[75] Successors: [Exit[23]]
NodeName: Exit[23] Successors: []
NodeName: UncheckedExceptions[29] Successors: [ParameterDeclaration[60], Exit[23]]
NodeName: ParameterDeclaration[60] Successors: [VarAccess[68]]
NodeName: VarAccess[68] Successors: [MethodAccess[69]]
NodeName: MethodAccess[69] Successors: [Exit[23]]
NodeName: NTACloseResource[53] Successors: [UncheckedExceptions[50]]
NodeName: UncheckedExceptions[50] Successors: [ParameterDeclaration[60], Exit[23]]

NodeName: Exit[23] Predecessor: [UncheckedExceptions[75], UncheckedExceptions[29], UncheckedExceptions[50], MethodAccess[69]]
NodeName: UncheckedExceptions[75] Predecessor: [MethodAccess[28]]
NodeName: MethodAccess[28] Predecessor: [MethodAccess[49]]
NodeName: MethodAccess[49] Predecessor: [VarAccess[48]]
NodeName: VarAccess[48] Predecessor: [ResourceDeclaration[27]]
NodeName: ResourceDeclaration[27] Predecessor: [ClassInstanceExpr[35]]
NodeName: ClassInstanceExpr[35] Predecessor: [Entry[16]]
NodeName: Entry[16] Predecessor: []
NodeName: UncheckedExceptions[29] Predecessor: [MethodAccess[28]]
NodeName: UncheckedExceptions[50] Predecessor: [NTACloseResource[53]]
NodeName: NTACloseResource[53] Predecessor: [MethodAccess[49]]
NodeName: MethodAccess[69] Predecessor: [VarAccess[68]]
NodeName: VarAccess[68] Predecessor: [ParameterDeclaration[60]]
NodeName: ParameterDeclaration[60] Predecessor: [UncheckedExceptions[29], UncheckedExceptions[50]]
